doc/examples/config/sharding.yaml

```diff
@@ -0,0 +1,39 @@
+credentials:
+  users:
+    guest:
+      roles: [super]
+
+iproto:
+  listen: 'unix/:./{{ instance_name }}.iproto'
+  advertise:
+    sharding: 'guest@'
```

Comment:
I think that we should leave a comment what does it actually mean.

Currently, it is not just advertise URI, but also an instruction for all master storage nodes to grant necessary privileges for this user. This is important details.

Reply:
BTW, I would add a separate user for this purpose. And it would be nice to set a password for it in the credentials section.

What doubts me: will not credentials applier revoke permissions granted by vshard at `config:reload()`?

Reply:
Added a new user. Also, grants will be revoked after reload().